docusaurus.config.js

```diff
@@ -41,14 +41,14 @@ module.exports = {
         path: "optimize",
         routeBasePath: "optimize",
         beforeDefaultRemarkPlugins: [versionedLinks],
```

Comment:
In previous version archivals, we were completely removing this Optimize docs configuration. This is the first version that included an extracted Optimize instance, so we need to retain this plugin definition from now on.

We do need to edit it, though, to only show the specific archived version.